.gas-snapshot

```diff
@@ -86,12 +86,12 @@ Market_setPriorityRestrictedThreshold:test() (gas: 157404)
 Market_setPriorityRestrictedThreshold:test_zeroAvailable() (gas: 152379)
 Market_setPurchasingTokenAndPriceMultiple:test() (gas: 1026562)
 Market_setPurchasingTokenAndPriceMultiple_revertsIfNotAdmin:test() (gas: 50791)
-Market_supplierSelectionUsingUpSuppliersLastRemoval:test() (gas: 918445)
+Market_supplierSelectionUsingUpSuppliersLastRemoval:test() (gas: 918444)
 Market_swapWithoutFee_emits_and_skips_transfer_when_transferring_wrong_erc20_to_rNori:test() (gas: 419376)
-Market_swap_emits_and_skips_transfer_when_transferring_wrong_erc20_to_rNori:test() (gas: 519931)
-Market_swap_emits_event_and_skips_mint_when_minting_rNori_to_nonERC1155Receiver:test() (gas: 597111)
-Market_swap_revertsWhenUnsafeERC20TransferFails:test() (gas: 216998)
-Market_validates_certificate_amount:test() (gas: 645513)
+Market_swap_emits_and_skips_transfer_when_transferring_wrong_erc20_to_rNori:test() (gas: 519930)
+Market_swap_emits_event_and_skips_mint_when_minting_rNori_to_nonERC1155Receiver:test() (gas: 597110)
+Market_swap_revertsWhenUnsafeERC20TransferFails:test() (gas: 216997)
+Market_validates_certificate_amount:test() (gas: 664841)
```

Comment:
Was surprised to see this big of a diff here. I'm guessing it's related to the fact that we've stuffed a bunch of setup logic into the test function. Emphasizes that we need too make sure we are putting as much of that kind of thing in `setUp` as possible, otherwise this report becomes much less useful


diff: 19328 (2.994%)